twitter/Models/User.m

```diff
@@ -0,0 +1,23 @@
+//
+//  User.m
+//  twitter
+//
+//  Created by Joey R. Hernandez Perez on 6/29/20.
+//  Copyright © 2020 Emerson Malca. All rights reserved.
+//
+
+#import "User.h"
+
+@implementation User
+
+-(instancetype) initWithDictionary:(NSDictionary *)dictionary {
```

Comment:
For extra challenge/fun I think it's nice to use class initializers. ex:

`Author *author = [Author fromDictionary:dictionary];`

reads better than 

`Author *author = [[Author alloc] initWithDictionary:dictionary];`

Just my 2c.